owasp-top10-2021-apps/a1/ecommerce-api/app/server.go

```diff
@@ -57,7 +57,11 @@ func main() {
 
 	echoInstance.GET("/", handlers.FormPage)
 	echoInstance.GET("/healthcheck", handlers.HealthCheck)
-	echoInstance.GET("/ticket/:id", handlers.GetTicket)
+
+	echoInstance.GET("/ticket/:id", handlers.GetTicket, middleware.JWTWithConfig(middleware.JWTConfig{
```

Comment:
Now that we are getting the `username` through the JWT. This `id` parameter could be removed.

Reply:
That `:id` field would still be useful if some user has multiple tickets. Or in the case of a superuser who has access to other user's tickets.
Anyway, breaking the API would be a different PR 😝

Reply:
That `:id` field refers to the `userID` not the `ticket_id` so I can't imagine it being useful if some user has multiple tickets. 🤓

Reply:
Oh, of course. Still applies the other reasons ;)
Thanks